pkgs/by-name/pr/preserves-tools/package.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchCrate }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "preserves-tools";
+  version = "4.992.2";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-1IX6jTAH6qWE8X7YtIka5Z4y70obiVotOXzRnu+Z6a0=";
+  };
+
+  cargoHash = "sha256-D/ZCKRqZtPoCJ9t+5+q1Zm79z3K6Rew4eyuyDiGVGUs=";
+
+  meta = {
+    description =
+      "Command-line utilities for working with Preserves documents.";
```

Comment:
```suggestion
    description = "Command-line utilities for working with Preserves documents";
```

Reply:
Can see my comment was marked as spam? But nevertheless, my point is that the punctuation at the end of the description should be removed to adhere to [this.](https://github.com/NixOS/nixpkgs/blob/3f83e9f16dcb04116e1564b1928e12252e0c1c8e/doc/stdenv/meta.chapter.md#description-var-meta-description)

Reply:
Sorry, I though you were being a whitespace developer. You are right and I've removed the comma.